### PR TITLE
init maxOfVector function

### PR DESCRIPTION
**Pull request by lubankp**
@lubankp added homework maxOfVector

### CODE REVIEW

**Comment by ziobron** [comment]
🏅 +1 XP for cooperation goes to @lubankp

**Comment by ziobron** [comment]
✅ CI Passed - Task: max-of-vector
🏅 Granted 5 XP for @lubankp

**Comment by ziobron** [comment]
⏱ 3 XP extra for @lubankp for on-time delivery

File: homework/max-of-vector/maxOfVector.hpp [hunk 1]
```diff
@@ -3,6 +3,12 @@
...
-    // TODO: Implement me :)
-    return {};
+    
+    int maximal = vec[0];
+
+    for(auto& element : vec )
+    if(element > maximal){
+    maximal = element;
+    }
+    return maximal;
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:13]: Add space after `if`, `for`, `while` `function` etc...

Add space between operator like `+` `-` `/` `:` `&&` etc..

File: homework/max-of-vector/maxOfVector.hpp [hunk 2]
```diff
@@ -3,6 +3,12 @@
 #include <limits>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
+    
+    int maximal = vec[0];
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:7]: What happen when vector is empty?

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**